test/e2e/diagram.overlays.rendering.test.ts

```diff
@@ -45,33 +44,72 @@ class ImageSnapshotThresholds extends MultiBrowserImageSnapshotThresholds {
   getFirefoxThresholds(): Map<string, ImageSnapshotThresholdConfig> {
     return new Map<string, ImageSnapshotThresholdConfig>([
       [
-        'overlays.start.flow.task',
+        'overlays.start.flow.task.gateway',
         {
-          linux: 0.0023, // 0.22444374007244416%
-          macos: 0.0025, // 0.24420815130801188%
-          windows: 0.007, // 0.6915678561497107%
+          linux: 0.0053, // 0.5229417116423329%
+          macos: 0.0061, // 0.6026399523082704%
         },
       ],
     ]);
   }
 }
 
+function buildOverlaySnapshotDir(config: MatchImageSnapshotOptions, position: OverlayPosition): string {
+  return join(config.customSnapshotsDir, `on-position-${position}`);
+}
+
+async function addOverlay(bpmnContainerElementHandle: ElementHandle<Element>, bpmnElementId: string, position: OverlayPosition): Promise<void> {
+  await page.fill('#bpmn-id-input', bpmnElementId);
```

Comment:
👍🏿 perfect to remove the `bpmnElementId`s previously hard coded in the js file loaded by the test page.